hackathon_site/dashboard/frontend/src/components/HOCs/withParticipantCheck/withParticipantCheck.test.js

```diff
@@ -60,9 +61,9 @@ describe("withAuthenticationCheck", () => {
 
         const store = mockStore(mockState);
 
-        const { queryByText } = render(
-            withStore(<WrappedComponent>{content}</WrappedComponent>, store)
-        );
+        const { queryByText } = render(<WrappedComponent>{content}</WrappedComponent>, {
+            store,
+        });
```

Comment:
some of them use the format
```
render(
<Wrapped>...,
{ store }
);
```
and some use 
```
render(<Wrapped>..., {
store,
});
```
do we want to be consistent with these?

Reply:
Do you just mean the formatting? That's left up to prettier based (mostly) on the line length, we have little control over it.